assets/styles/authoring/image-upload.scss

```diff
@@ -0,0 +1,37 @@
+@import 'delivery/variables';
+
+.drag-and-drop-zone {
+  background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' rx='13' ry='13' stroke='%23C8C8C8FF' stroke-width='3' stroke-dasharray='7%2c 16' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");
+  border-radius: 13px;
+  background-color: $body-bg;
+}
+
+.on-drag {
+  background-color: darken($color: $body-bg, $amount: 10%);
+}
+
+.btn-tertiary {
+  margin: auto;
+  display: block;
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+}
+
+.img-input-file {
+	opacity: 0;
```

Comment:
NIT: indent

Reply:
Could it be a github error? Seeing it fine locally